releng/third-party/pom.xml

```diff
@@ -116,25 +117,41 @@
 									<id>org.adoptopenjdk:jemmy-swt:${jemmy.version}</id>
 								</artifact>
 								<artifact>
-									<id>org.eclipse.jetty.websocket:websocket-jetty-server:${jetty.version}</id>
+									<id>org.eclipse.jetty.websocket:jetty-websocket-jetty-server:${jetty.version}</id>
 								</artifact>
 								<artifact>
-									<id>org.eclipse.jetty.websocket:websocket-javax-server:${jetty.version}</id>
+									<id>org.eclipse.jetty.ee9:jetty-ee9-servlet:${jetty.version}</id>
+								</artifact>
```

Comment:
There's some interesting formatting going on in this file, some empty lines and incorrectly tabbing